### DISCUSSION TITLE
Setting up a carousel

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [fancyapps/ui#612] [Q&A]
Hello)) I noticed that there is a demo on the site "Carousel Tween Example #3" - I would like to know if it is possible to configure the carousel in the same way, only so that there are 3 active elements, not 1.
Like on a picture. Suddenly this slider is capable of this.

![test](https://github.com/fancyapps/ui/assets/63208219/d450ea10-f464-4b40-a640-f95ff0439725)

**Comment by fancyapps** [comment]
Hi,

Here is one possible solution - https://stackblitz.com/edit/js-cbcxrg?file=index.html,index.js,style.css

**Reply by MyNameTony1000** [reply]
Hello, thanks for the quick response and for the demonstration)) I was a little lost)) As far as I understand, there is no ability to scroll through one element and maintain equal indents between all elements all the time - right?) Since the slider needs to be constantly recalculated.

**Reply by fancyapps** [reply]
Hi, Sorry, but what did you mean by "there is no ability to scroll" and "equal indents between all elements all the time"?

**Reply by MyNameTony1000** [reply]
Hello, as far as I understand, it is now done so that the slides are grouped by 3 pieces and scrolled by 3 pieces. I’m asking - is it possible to scroll through the slider by 1 element, and not immediately by 3? Regarding the indents - now they are the same, but due to the fact that visual scale is used, the indents between groups of slides are greater than the indents between the slides themselves in the group.
The question was: is it possible to make a slider like in the picture, only so that it scrolls through 1 element (square) and so that in all places there are the same indents visually (as in the picture) ?
